mlflow/pyfunc/loaders/__init__.py

```diff
@@ -0,0 +1 @@
+import mlflow.pyfunc.loaders.chat_model  # noqa: F401
```

Comment:
is this necessary? or does python load all files in the subdirectory into the module by default?

Reply:
Python doesn't. If we want to do `from mlflow.pyfunc.loaders import chat_model`, we need this line, otherwise we don't.